### PR TITLE
[CertChecker] install WWDR certificate if not already installed

### LINKED ISSUE #36: Could not find the newly generated certificate installed (RuntimeError) if WWDR Intermediate is missing
Title says it all, just noticed that cert fails if the WWDR intermediate certificate is not installed. This basically leads to security/Mac OS determining that the just created identity is not signed by a known authority, ultimately making cert fail. 

    [23:12:42]: Couldn't find an existing certificate... creating a new one
    1 key imported.
    1 certificate imported.
    [23:12:44]: Successfully generated asdfasdf which was imported to the local machine.
    /Users/xxx/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cert-0.3.3/lib/cert/cert_runner.rb:7:in `launch': Could not find the newly generated certificate installed (RuntimeError)

### PR DESCRIPTION
**Pull request by jpsim**
This should resolve fastlane/cert#36. I'm happy to write some tests for this if you point me to the appropriate location to do so @KrauseFx.

### CODE REVIEW

File: lib/fastlane_core/cert_checker.rb [hunk 1]
```diff
@@ -10,6 +10,20 @@ def self.installed?(path)
...
 
+    def self.wwdr_certificate_installed?
+      certificate_name = "Apple Worldwide Developer Relations Certification Authority"
+      `security find-certificate -c #{certificate_name}`
+      $?.success?
+    end
+
+    def self.install_wwdr_certificate
+      Dir.chdir '/tmp'
+      url = 'http://developer.apple.com/certificationauthority/AppleWWDRCA.cer'
```
Comment (KrauseFx) [inline lib/fastlane_core/cert_checker.rb]: This should be `https`

**Comment by KrauseFx** [comment]
Really sorry for the delay, I somehow lost the tab. Looks good - just the URL needs to be updated. I'll merge and change it directly.

**Comment by KrauseFx** [comment]
Have you tested it on your machine or do I need to do additional tests on a clean machine?

**Comment by jpsim** [comment]
> Have you tested it on your machine or do I need to do additional tests on a clean machine?

It's only been 4 days, but I'm afraid I don't remember. I tried lots of things, and ended up going with a fastlane-less route in the end.

File: lib/fastlane_core/cert_checker.rb [hunk 2]
```diff
@@ -10,6 +10,20 @@ def self.installed?(path)
       return ids.include? finger_print
     end
 
+    def self.wwdr_certificate_installed?
+      certificate_name = "Apple Worldwide Developer Relations Certification Authority"
+      `security find-certificate -c #{certificate_name}`
```
Comment (KrauseFx) [inline lib/fastlane_core/cert_checker.rb]: The missing `"` around the certificate name were the problem :+1:

Comment (lkraider) [inline lib/fastlane_core/cert_checker.rb]: This looks in the current security search list, while the import command directly specifies the `login.keychain`. This causes issues when the search list was changed to a custom keychain, see https://github.com/fastlane/sigh/issues/264 .

**Pull request merged**